ngraph/frontend/paddlepaddle/src/frontend.cpp

```diff
@@ -127,7 +119,7 @@ namespace ngraph
                 parameter_nodes.push_back(param);
             }
 
-            const auto& op_places = model->getOpPlaces();
+            const auto& op_places = model->returnOpPlaces();
```

Comment:
Can we probably do `determine_cut_nodes` here and use 'const' methods after this deferred update?

Reply:
We can, but it makes more sense to have it done internally.